src/components/transitions/row-expand-transition.js

```diff
@@ -7,26 +7,23 @@ export default {
     addOnceEventListener(el, 'transitionend', done)
 
     // Get height that is to be scrolled
-    const height = el.dataset.height || el.clientHeight
-    el.dataset.height = height
     el.style.overflow = 'hidden'
     el.style.height = 0
     el.td.style['border-bottom'] = '1px solid rgba(0,0,0,0.12)'
 
-    setTimeout(() => {
-      el.dataset.height = el.style.height = `${el.scrollHeight}px`
-    }, 50)
+    setTimeout(() => el.style.height = `${el.scrollHeight}px`, 0)
```

Comment:
I'm surprised the linter did not bark about this. Functions should never return assignment.

``setTimeout(() => (el.style.height = `${el.scrollHeight}px`), 0)``